syntax = "proto3";

package weather;

import "common.proto";

message WeatherRequest {
  common.Location location = 1;
}

message Weather {
  double temperature_c = 1;
  string description = 2;
}

service WeatherService {
  rpc GetWeather(WeatherRequest) returns (Weather);
}

